_data/projects/covid19app.yml

```diff
@@ -0,0 +1,11 @@
+---
+name: Covid 19 App
+desc: Covid 19 Stastics app, made in Android with JAVA
+site: https://github.com/jayskhatri/covid19-app
+tags:
+- covid19statistics
+- android
+- design
+upforgrabs:
+  name: good first issue
+  link: https://github.com/jayskhatri/covid19-app/labels/good%20first%20issue
```

Comment:
@jayskhatri thanks for resolving the changes needed. It looks like this label doesn't have any issues assigned to it, which means it won't be listed on the site once we start retrieving stats. 

Were you interested in adding some tasks available for new contributors?

Reply:
Oopss sorry😬

Actually while solving previous errors i removed this label from one issue, now it's added. Thanks for pointing it out.